docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md

```diff
@@ -317,29 +317,25 @@ For MySQL configuration instructions, see [Set up MySQL database](#set-up-mysql-
 
 3. In the Firebase console, click **Add project**.
 
-4. Enter your project name.
+4. Enter your project name and click **Continue**.
 
-5. You may choose to enable Google Analytics for your project (optional).
-
-6. Click **Continue**.
-
-7. Click **Create a project**.
+5. Click **Create project**.
 
 ### Create a Firebase config
 
 1. In your Firebase project console, click on the **settings icon** that resembles a gear in the top left corner.
 
 2. Select **Project settings** from the dropdown menu.
 
-3. Under the **General** tab, scroll down to **Your apps** and select the **web** platform.
+3. Under the **General** tab, scroll down to **Your apps** and click on the **code icon** to select the web platform.
```

Comment:
3. Under the **General** tab, scroll down to **Your apps** and click on the **code icon** (</>) to select the web platform.